src/quickstart/provider/aws/boot.md

```diff
@@ -1,263 +1,365 @@
-# Booting Kubernetes on Amazon Elastic Compute
+# Booting Kubernetes on Amazon Elastic Compute with kops
 
 Amazon Elastic Compute Cloud (Amazon EC2) is a web service that provides compute capacity in the cloud. This quickstart
-guide uses AWS EC2 to boot a Kubernetes cluster using the open source provisioning scripts.
+guide uses AWS EC2 to boot a Kubernetes cluster using [kubernetes kops](https://github.com/kubernetes/kops).
 
-## Pre-requisites
 
-1. You need an active AWS account. Visit [AWS portal](http://aws.amazon.com) to sign up
-2. You need AWS API keys with full access
-3. Install the AWS cli tools, you can find instructions for your platform at [AWS Command Line Interface Site](https://aws.amazon.com/cli/)
+## Installing kops
 
-To verify that your CLI is configured properly, run `aws ec2 describe-regions`:
+Download the [latest](https://github.com/kubernetes/kops/releases/latest) version of kops
 
+Note: We will be using `kops` 1.5.0-beta2 in this example.
+
+#### macOS
```

Comment:
So I think Sierra calls it `macOS` .. instead of the traditional `OS x`

Sound good?